src/components/shared/Navbar.tsx

```diff
@@ -62,6 +62,9 @@ export const Navbar = ({
           <MenuItem to="/docs/quick-installation/">
             GET STARTED
           </MenuItem>
+          <MenuItem to="/docs/quick-installation-new-operator/">
+            GET STARTED V2
+          </MenuItem>
```

Comment:
```suggestion
```
Just to clarify, on the homepage we need to have a single `GET STARTED` reference to prevent confusion for new users.  Ideally, we will be using the DataScienceCluster blog in #167 to explain was `v2` is and direct users to the `v2` installation until the new DataScienceCluster deployment supports deployment of all of the existing ODH components